packages/html/src/grid/tests/grid-filter-menu-angular.tsx

```diff
@@ -1,204 +1,201 @@
-                                        <tr className="k-table-row">
-                                            <td className="k-table-td k-touch-action-auto" colSpan={1}><Checkbox /></td>
-                                            <td className="k-table-td k-touch-action-auto" colSpan={1}>Product1</td>
-                                            <td className="k-table-td k-touch-action-auto k-command-cell" colSpan={1}><Button icon="trash">Delete</Button></td>
-                                        </tr>
-                                        <tr className="k-table-row k-table-alt-row k-alt">
-                                            <td className="k-table-td k-touch-action-auto" colSpan={1}><Checkbox /></td>
-                                            <td className="k-table-td k-touch-action-auto" colSpan={1}>Product2</td>
-                                            <td className="k-table-td k-touch-action-auto k-command-cell" colSpan={1}><Button icon="trash">Delete</Button></td>
-                                        </tr>
-                                    </tbody>
-                                </table>
-                            </div>
-                            <div className="k-height-container">
-                                <div></div>
-                            </div>
+                        <div className="k-height-container">
+                            <div></div>
                         </div>
-                    </div>
-                </div>
-            </div>
+                    </GridContent>
+                </GridContainer>
+            </Grid>
+
             <h3>Sortable &amp; Resizable</h3>
-            <div dir="ltr" className="k-grid k-grid-md">
-                <div className="k-grid-aria-root" role="grid">
-                    <div className="k-grid-header">
-                        <div className="k-grid-header-wrap">
-                            <table className="k-table k-table-md k-grid-header-table">
+            <Grid dir="ltr" _renderAriaRoot>
+                <GridHeader>
+                    <div className="k-grid-header-wrap">
+                        <GridHeaderTable>
+                            <colgroup>
+                                <col style={{ width: '50px' }} />
+                                <col />
+                                <col />
+                            </colgroup>
+                            <TableThead>
+                                <TableRow>
+                                    <GridHeaderCell resizable rowspan={1} colspan={1}>
+                                        <Checkbox />
+                                    </GridHeaderCell>
+                                    <GridHeaderCell resizable rowspan={1} colspan={1} sortable menu="filter" columnTitle="Name"></GridHeaderCell>
+                                    <GridHeaderCell resizable rowspan={1} colspan={1} columnTitle="Command"></GridHeaderCell>
```

Comment:
Is this cell resizable on purpose?

Reply:
Yes, there was a mistake in the original test, the example was titled as resizable however there was no column-resizer, hence the resizable cell